.github/workflows/build_and_test.yml

```diff
@@ -87,6 +87,7 @@ jobs:
             sparkr=`./dev/is-changed.py -m sparkr`
             tpcds=`./dev/is-changed.py -m sql`
             docker=`./dev/is-changed.py -m docker-integration-tests`
+            k8s=`./dev/is-changed.py -m kubernetes`
```

Comment:
I believe we should enable k8s test always instead of this. `core` module changes will affect many scheduling tests too.

Reply:
FYI, historically, log4j2 change also causes K8s integration test failure due to `log4j2.properties` file changes.

Reply:
OK, I will change this soon.